servers/physics_3d/soft_body_3d_sw.cpp

```diff
@@ -941,6 +982,10 @@ void SoftBody3DSW::_compute_area_gravity(const Area3DSW *p_area) {
 	}
 }
 
+Vector3 SoftBody3DSW::_compute_area_windforce(const Area3DSW *p_area, const Face *p_face) {
+	return p_area->get_wind_force_magnitude() * 0.33333333333 * p_face->ra * vec3_dot(p_face->normal, p_area->get_wind_direction()) * pow(vec3_dot(p_face->centroid - p_area->get_wind_source(), p_area->get_wind_direction()), p_area->get_wind_attenuation_factor()) * p_face->normal;
```

Comment:
This could be split in several lines with local variables to make it more readable.